src/cdev/commands/local_development.py

```diff
@@ -5,21 +5,21 @@
 from cdev.default.output_manager import CdevOutputManager
 
 
-def develop_command_cli(args):
+def develop_command_cli(args) -> None:
     config = args[0]
     set_global_logger_from_cli(config.loglevel)
     develop_command(args)
 
 
-def develop_command(args):
+def develop_command(args) -> None:
 
     output_manager = CdevOutputManager()
     myProject = Project.instance()
 
     print(f"Live Development")
 
 
-
+# ANIBAL dead code
```

Comment:
This is some code that I had written to do live reloading of the current state of a project and automatically deploy a project. Since then, I have refactored the architecture a little bit, so I just commented out this code that way I don't forget where it is. 

We can remove it in a specific commit and remember that commit because it might be helpful to relook at this when we come back to reimplement the feature.